files/www/cgi-bin/mesh

```diff
@@ -298,7 +299,31 @@ print "<center>\n";
 alert_banner();
 
 # page header
-print "<h1>$node mesh status</h1><hr><nobr>";
+print "<h1>$node mesh status</h1>";
+
+if(-f "/etc/latlon") {
+    $rcgood=open(FILE, "/etc/latlon");
+    push @errors, "ERROR: reading lat/lon data\n" unless $rcgood;
+    while(<FILE>){
+        chomp;
+        push @lines,$_;
+    }
+    close(FILE);
+    $lat=$lines[0];
+    $lon=$lines[1];
+    print "<center><strong>Location Info</strong> $lat $lon</center>";
+} else {
+    print "<center><strong>Location Info Not Available</strong></center>";
+}
+print "<table id='node_description_display'><tr><td>$node_desc</td></tr></table>";
+
+print "<hr><nobr>";
+
+print "<table><th align='right'>OLSR Entries</th><td><nobr>Total = " .                                                                                                       
+ `ip route list table 30 | wc -l` .                                                                                          
```

Comment:
i would prefer that the commands be performed earlier in the code and saved as a var, then, referenced here in the output.   also, $(cmd) here too